### ISSUE TITLE
Jsr310Converters does not support ZonedDateTime [DATACMNS-834]

### ISSUE THREAD

**Issue by spring-projects-issues** [spring-projects/spring-data-commons#1296]
**[Tim B](https://jira.spring.io/secure/ViewProfile.jspa?name=boeckstaenstim@gmail.com)** opened **[DATACMNS-834](https://jira.spring.io/browse/DATACMNS-834?redirect=false)** and commented

I was trying to store an entity with a ZonedDateTime field to MongoDB using Spring Data Mongodb 1.8.4. This resulted in an error. The converters from the class Jsr310Converters are loaded but this class does not support ZonedDateTime. When I add custom converters for this type everything works.
I have found this comment on the original issue to add JSR310 support. [comment here](https://jira.spring.io/browse/DATACMNS-411?focusedCommentId=98391&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-98391) Which states only the new time types are supported that Spring can convert a Calendar to. I couldn't find the Spring class that converts time classes to Calendar so I don't really know whether it is correct that ZonedDateTime is ommitted.
So bottom line: Could ZonedDateTime support be added to Jsr310Converters or not?

---

**Affects:** 1.11.4 (Gosling SR4)

**Issue Links:**
- [DATAMONGO-1468](https://jira.spring.io/browse/DATAMONGO-1468) QueryDsl repository between predicates with ZonedDateTime run into CodecConfigurationException

- [DATAJPA-1242](https://jira.spring.io/browse/DATAJPA-1242) AuditAware not converting to ZonedDateTime anymore
